gr-zeromq/lib/pub_sink_impl.h

```diff
@@ -28,6 +28,8 @@ class pub_sink_impl : public pub_sink, public base_sink_impl
                   bool pass_tags,
                   int hwm);
 
+    ~pub_sink_impl();
```

Comment:
Since these methods will be empty, will putting them entirely here work for the issue at hand? If so, let's do that ... the code is simpler and cleaner.

Reply:
Shouldn't this destructor be virtual? Otherwise it will override the destructors of the base classes.

Reply:
My understanding of OOP, classes and destructors is quite limited, so I just oriented myself on other zeromq blocks with existing destructors.